networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -24,6 +24,15 @@ For an additional network, IP addresses are provisioned through an IP Address Ma
 
 * Applying a YAML manifest: You can manage the additional network directly by creating an `NetworkAttachmentDefinition` object. This approach allows for the chaining of CNI plugins.
 
+[NOTE]
+====
+When deploying {product-title} nodes with the multiple network interfaces on {rh-openstack-first} with OVN SDN, subnet id of the secondary interface might take precedence over the subnet id of the primary interface. In this case, remove the DNS nameserver of the secondary interface's subnet id:
+[source,terminal]
+----
+$ openstack subnet set --no-dns-nameservers <subnet_id>
```

Comment:
I wonder if that's enough, of if you need to explicitly set the DNS nameserver to 0.0.0.0 like @itzikb-redhat was suggesting.
@itzikb-redhat can you confirm?

Reply:
> I wonder if that's enough, of if you need to explicitly set the DNS nameserver to 0.0.0.0 like @itzikb-redhat was suggesting. @itzikb-redhat can you confirm?

If I understand correctly from https://docs.openstack.org/neutron/rocky/admin/config-dns-res.html it should be the same. I checked with setting to 0.0.0.0

Reply:
@itzikb-redhat @mandre Do we need to change the command? `$ openstack subnet set --no-dns-nameservers <subnet_id>`

Reply:
I would prefer to set it to 0.0.0.0